journalpump/journalpump.py

```diff
@@ -551,6 +552,18 @@ def _validate_and_build_secret_filters(self, config):
 
         return secret_filters
 
+    def _configure_threshold_for_metric_emit(self, config):
+        threshold_for_metric_emit = config.get("threshold_for_metric_emit")
+        if threshold_for_metric_emit is not None:
+            try:
```

Comment:
Isn't this the same as having no try/except block and just doing `threshold = int(threshold_for_metric_emit)`?

Reply:
yep, I was just supposing some non-sense input like a string like "A", "B" was put as the config.

Reply:
Oh, yes, I got your point, I didn't add much extra info to the raise, so no need to do a try except here